src/locales/ko/mystery-encounters/weird-dream-dialogue.json

```
@@ -1,22 +1,22 @@
 {
-  "intro": "A shadowy woman blocks your path.\nSomething about her is unsettling...",
-  "speaker": "Woman",
-  "intro_dialogue": "I have seen your futures, your pasts...$Child, do you see them too?",
+  "intro": "어두운 실루엣의 여자가 앞을 막아선다.\n그녀에게서 뭔가 불안한 느낌이 든다…….",
```

```suggestion
  "intro": "어떤 여자의 실루엣이 앞을 막아섰다.\n그녀에게서 뭔가 불안한 느낌이 든다…….",
```
"실루엣" 자체에 어두운 윤곽이라는 뜻이 있다 보니 "실루엣"이라는 단어를 쓴다면 "어두운"은 넣지 않아도 좋을 듯합니다.